src/scf-release/packages/suse-fuse/packaging

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e -x
+
+DST=${BOSH_INSTALL_TARGET}
+SRC=/var/cache/zypp/packages
+
+# Clear the rpm cache. We want to see later only new files.
+zypper clean
+
+# Get fuse rpm and dependencies
+zypper --non-interactive --quiet install --download-only fuse
+
+# Get and copy the content of the rpm cache into the package-to-be.
+cp $(find /var/cache/ -type f -name '*.rpm') ${DST}/
```

Comment:
Oops. The directory should be `${SRC}`

Reply:
Done with commit 838d12fad5710bd0aaa9d0a107f1fe1b0688a90f, then superseded in commit 571a972ea60268849419855b642ed8be94ef47fa